dev/CommandBarFlyout/CommandBarFlyout.cpp

```diff
@@ -14,6 +14,26 @@
 bool CommandBarFlyoutTrace::s_IsDebugOutputEnabled{ false };
 bool CommandBarFlyoutTrace::s_IsVerboseDebugOutputEnabled{ false };
 
+// List of AppBarButton dependency properties being listened to for raising the CommandBarFlyoutCommandBar::OnCommandBarElementDependencyPropertyChanged notifications.
+const winrt::DependencyProperty CommandBarFlyout::s_appBarButtonDependencyProperties[s_commandBarElementDependencyPropertiesCount]
+{
+    winrt::AppBarButton::IconProperty(),
+    winrt::AppBarButton::IsCompactProperty(),
```

Comment:
Does this and LabelPosition actually affect the sizing?  I don't believe they affect AppBar*Buttons that are in the secondary commands.

Reply:
Ah, my new testing showed that LabelPosition and IsCompact have no effect on AppBarButton and AppBarToggleButton as secondary commands. I took them out of the list.  Icon does have an effect and is kept. Thanks!